crates/symbolicator/src/services/download/mod.rs

```diff
@@ -53,8 +55,10 @@ pub enum DownloadError {
     Sentry(#[from] sentry::SentryError),
     #[error("failed to fetch data from S3")]
     S3(#[from] s3::S3Error),
-    #[error("S3 error code: {1} (http status: {0})")]
-    S3WithCode(StatusCode, String),
+    #[error("aws-sdk: failed to fetch data from S3")]
+    S3SDK(#[from] aws_smithy_http::byte_stream::Error),
+    // #[error("S3 error code: {1} (http status: {0})")]
+    // S3WithCode(StatusCode, String),
```

Comment:
should probably be removed

Reply:
Removed; I left it as a reminder that it's possible to report the HTTP status code using the `from` macro, although the mechanics of how aren't clear to me yet.